Makefile

```diff
@@ -43,21 +43,21 @@ endif
 
 .DEFAULT_GOAL := all
 all: show-rules
-	@echo "Please use sub-rules to build HolBA."
+	@echo 'Please use sub-rules to build HolBA.'
 
 show-rules:
-	@echo "Available rules:"
-	@echo "     - Holmakefiles: generates \`Holmakefile\`s from \`Holmakefile.gen\` files."
+	@echo 'Available rules:'
+	@echo '     - Holmakefiles: generates `Holmakefile`s from `Holmakefile.gen` files.'
 ifdef HOLBA_HOLMAKE_AVAILABLE
-	@echo "     - theory: builds only src/theory"
-	@echo "     - main: builds HolBA, but without the examples or documentation"
-	@echo "     - tests: builds HolBA and runs all the tests"
-	@echo "     - examples-base: builds HolBA and the examples for each tool"
-	@echo "     - examples-all: builds HolBA and all the examples (base + HOLBA/examples/)"
-	@echo "     - benchmarks: builds HolBA and all the benchmarks"
+	@echo '     - theory: builds only src/theory/'
+	@echo '     - main: builds HolBA, but without the examples or documentation'
+	@echo '     - tests: builds HolBA and runs all the tests'
+	@echo '     - examples-base: builds HolBA and the examples for each tool'
+	@echo '     - examples-all: builds HolBA and all the examples (base + HolBA/examples/)'
```

Comment:
I wrote it on purpose in capital, but it isn't consistent everywhere anyways. So maybe this is better.